package.json

```diff
@@ -37,6 +37,7 @@
     "@chakra-ui/react": "1.8.8",
     "@emotion/react": "^11",
     "@emotion/styled": "^11",
+    "@emurgo/cardano-serialization-lib-browser": "10.2.0",
```

Comment:
We use [this library](https://github.com/dcSpark/cardano-multiplatform-lib) now

Reply:
@MartinSchere fixed